builder/builder.go

```diff
@@ -572,36 +596,112 @@ func (b *Builder) BuildUpdate(prefixflag string, minVersion int, format string,
 		fmt.Println("Signed Manifest.MoM")
 	}
 
+	fullChrootDir := filepath.Join(b.Statedir, "image", b.Mixver, "full")
+	if _, err = os.Stat(fullChrootDir); err != nil {
+		return errors.Wrapf(err, "couldn't access the full chroot")
+	}
+
+	outputDir := filepath.Join(b.Statedir, "www")
```

Comment:
We are ignoring the output directory specified in server.ini again. I don't think this is a big deal right now, but after we have the swupd code fully in mixer we should consolidate configuration files and access *one* configuration in *one* place.

Reply:
Mixer itself ignores that configuration, so I suggest we handle this consolidation in separate (possibly after we get things working).

Reply:
+1

Reply:
+1 kill the serveri.ini